src/command/command.js

```diff
@@ -133,6 +134,34 @@ yargs
                 await exporter.export(argv.flotiqApiKey, argv.directory, true);
             }
         })
+    .command('sdk install [language] [directory] [flotiqApiKey]', 'Install Flotiq SDK', (yargs) => {
+        yargs.positional('language', {
+            describe: 'SDK language',
+            type: 'string',
```

Comment:
Can we use enum or validate if user provided values from readme (csharp, go, java, javascript, php, python, typescript)?